comm.cpp

```diff
@@ -213,6 +228,20 @@ int Endpoint::read_msg(struct buffer *pbuf)
     if (_crc_check_enabled && !_check_crc())
         return 0;
 
+    msg_entry = mavlink_get_msg_entry(msg_id);
+    if (msg_entry == nullptr) {
+        log_error("No message entry for %u", msg_id);
+    }
+
+    *target_sysid = -1;
+    *target_compid = -1;
+
+    if (msg_entry->flags & MAV_MSG_ENTRY_FLAG_HAVE_TARGET_SYSTEM)
+        *target_sysid = payload[msg_entry->target_system_ofs];
+
+    if (msg_entry->flags & MAV_MSG_ENTRY_FLAG_HAVE_TARGET_COMPONENT)
+        *target_compid = payload[msg_entry->target_component_ofs];
```

Comment:
I don't mind the missing `{ }` on if statements with a single line. However be consistent with what is there.... Just some lines above you used braces